src/scripts/core/models/grid.ts

```diff
@@ -8,33 +8,44 @@ gsap.registerPlugin(MotionPathPlugin);
 import { Curve } from "../services/curve"
 
 export class Grid extends Container {
-
-    private _availWidth: number;
+    private _background: Sprite = new Sprite(Assets.get('gridBackground'));
+    private _columnContainer = new Container();
```

Comment:
Generally I wouldn't recommend 'newing' these here, this is functionally similar (but not the same) as 'newing' them in the constructor. Most importantly you don't have complete control over the order in which they are created here. Generally it's a good idea to only assign when declaring if it has a simple default, such as a number or string.